manual/cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/WhereTest.scala

```diff
@@ -105,66 +105,66 @@ class WhereTest extends DocumentingTestBase {
       assertions = (p) => assertEquals(List(node("Peter")), p.columnAs[Node]("n").toList))
   }
 
   @Test def string_matching_ends_with() {
     testQuery(
       title = "Match the end of a string",
       text = """The `ENDS WITH` operator is used to perform case-sensitive matching on the end of strings.""".stripMargin,
-      queryText = """match (n) where n.name ENDS WITH 'ter' return n""",
-      optionalResultExplanation = """"+Peter+" will be returned because his name ends with `ter`.""",
+      queryText = """MATCH (n) WHERE n.name ENDS WITH 'ter' RETURN n""",
+      optionalResultExplanation = """"Peter" will be returned because his name ends with `ter`.""",
       assertions = (p) => assertEquals(List(node("Peter")), p.columnAs[Node]("n").toList))
   }
 
   @Test def string_pattern_matching_negation() {
     testQuery(
       title = "String matching negation",
       text = "Use the `NOT` keyword to exclude all matches on given string from your result:",
-      queryText = """match (n) where NOT n.name ENDS WITH 's' return n""",
-      optionalResultExplanation = """"+Peter+" will be returned because his name does not end with `s`.""",
+      queryText = """MATCH (n) WHERE NOT n.name ENDS WITH 's' RETURN n""",
+      optionalResultExplanation = """"Peter" will be returned because his name does not end with `s`.""",
       assertions = (p) => assertEquals(List(node("Peter")), p.columnAs[Node]("n").toList))
   }
 
   @Test def string_matching_contains() {
     testQuery(
       title = "Match anywhere in a string",
       text = """The `CONTAINS` operator is used to perform case-sensitive matching regardless of location in strings.""".stripMargin,
-      queryText = """match (n) where n.name CONTAINS 'ete' return n""",
-      optionalResultExplanation = """"+Peter+" will be returned because his name contains `ete`.""",
+      queryText = """MATCH (n) WHERE n.name CONTAINS 'ete' RETURN n""",
+      optionalResultExplanation = """"Peter" will be returned because his name contains `ete`.""",
       assertions = (p) => assertEquals(List(node("Peter")), p.columnAs[Node]("n").toList))
   }
 
   @Test def has_property() {
     testQuery(
       title = "Property exists",
-      text = "Use the `EXISTS()` function to only include nodes or relationships in which a property exists.",
-      queryText = """match (n) where exists(n.belt) return n""",
+      text = "Use the `exists()` function to only include nodes or relationships in which a property exists.",
+      queryText = """MATCH (n) WHERE exists(n.belt) RETURN n""",
       optionalResultExplanation =
-        """"+Andres+" will be returned because he is the only one with a `belt` property.
+        """"Andres" will be returned because he is the only one with a `belt` property.
           |
           |[IMPORTANT]
-          |The `HAS()` function has been superseded by `EXISTS()` and has been removed.
+          |The `has()` function has been superseded by `exists()` and has been removed.
```

Comment:
I'm wrong; it was _deprecated_ in 2.3 and removed in 3.0. The changelog doesn't seem to include this, however!